packages/strapi-design-system/src/v2/MainNav/NavBrand.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { useMainNav } from './MainNavContext';
+import { BaseLink, BaseLinkProps } from '../../BaseLink';
+import { Box } from '../../Box';
+import { Flex } from '../../Flex';
+import { Typography } from '../../Typography';
+import { VisuallyHidden } from '../../VisuallyHidden';
+
+const BrandIconWrapper = styled.div<{ condensed?: boolean }>`
+  border-radius: ${({ theme }) => theme.borderRadius};
+
+  svg,
+  img {
+    height: ${({ condensed }) => (condensed ? `${40 / 16}rem` : `${32 / 16}rem`)};
+    width: ${({ condensed }) => (condensed ? `${40 / 16}rem` : `${32 / 16}rem`)};
+  }
+`;
+
+const NavLinkWrapper = styled(BaseLink)`
+  text-decoration: unset;
+  color: inherit;
+`;
+
+export interface NavBrandProps extends BaseLinkProps {
+  icon: React.ReactNode;
+  title: string;
+  to?: string;
+  workplace: string;
+}
+
+export const NavBrand = React.forwardRef<HTMLAnchorElement, NavBrandProps>(
+  ({ workplace, title, icon, ...props }, ref) => {
+    const condensed = useMainNav();
+    props.to = props?.to ?? '/';
```

Comment:
```suggestion
   /**
   * TODO: this shouldn't be here, because we're assuming you're
   * passing a ReactRouter Link which isn't necessarily the case.
   */
    props.to = props?.to ?? '/';
```

I know you didn't write this per say, but i want the comment so we remember to not do this 😅